tests/helpers/acceptance.js

```diff
@@ -55,6 +58,18 @@ function createTestTargets(projectName, options) {
   options.command = options.command || 'new';
 
   return applyCommand(options.command, projectName, '--skip-npm', '--skip-bower', `--directory=${outputDir}`)
+    .then(() => {
+      if (options.createESLintConfig) {
```

Comment:
Hrm. We just need to do this _after_ `linkDependencies` meanwhile you were doing it at the `createTestTargets` point. I ... stand corrected. I'd like to make it actually run the blueprints, there's no particular reason that `linkDependencies` needs to be run `beforeEach` that I can tell in a quick glance while correcting my previous comment...